carton/file.py

```diff
@@ -27,7 +27,11 @@ def __call__(self, *paths):
     def norm(self):
         return self.__class__(normalize_path(self))
 
+    def __truediv__(self, *paths):
```

Comment:
Based on the [document](https://docs.python.org/3/reference/datamodel.html#object.__truediv__) of Python 3, I think `__truediv__` should accept exactly single argument.